### DISCUSSION TITLE
How to define environment variables for `bazel run rust_binary`?

### DISCUSSION THREAD

**Discussion by tamuhey** [bazelbuild/rules_rust#949] [Q&A]
I want to define environment variables (e.g. `RUST_LOG=debug`) for `bazel run :myapp` where `:myapp` is a `rust_binary`.
How to add vars?

**Comment by UebelAndre** [comment]
I don't think there's a way to set environment variables on the target but with `bazel run` commands, you can always set the environment before yourself.
```bash
RUST_LOG=debug bazel run //:myapp
```
